atc/db/build.go

```diff
@@ -1185,6 +1259,27 @@ func (b *build) saveEvent(tx Tx, event atc.Event) error {
 
 func createBuild(tx Tx, build *build, vals map[string]interface{}) error {
 	var buildID int
+
+	// See if the pipeline has credential managers configured, ff yes, add to build.
```

Comment:
In some case, a pipeline credential manager may change. After the change, if there are still pending builds, then old builds should still use old credential manager. That's reason why I copy the info from `pipelines` to `builds`.

Reply:
:thinking: Do you feel that this will be a common occurrence worth accounting for? I feel like I wouldn't be too shocked if, as a user, I updated the credential manager config and my older in-flight builds adopted the new changes. For example, maybe I just rotated my Vault credentials. Do you expect something different from your workflow?

Reply:
Actually I don't have strong opinion on this. Updating Vault configure should not be something happening now and again. I can remove `var_sources` from `builds`.

Reply:
Removed `var_sources` from `builds`.